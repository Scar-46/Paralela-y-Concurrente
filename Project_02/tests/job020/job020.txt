plate401.bin 1 45.8 272.7 0.0000015
plate411.bin 15 50 291 0.000009
plate405.bin 1 57.4 777 0.000007
plate433.bin 10 17.6 700 0.00071
plate411.bin 1200 92.44 750 0.00005
plate422.bin 60 111.3 25 0.0065
plate433.bin 60 34.7 375 0.0019
plate405.bin 30 61.31 100 0.000007
plate422.bin 60 111.3 300 0.0065
plate422.bin 15 22.1 471.75 0.000085
plate433.bin 1 23 400 0.002
