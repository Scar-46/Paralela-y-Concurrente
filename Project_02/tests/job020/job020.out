plate401.bin	1	45.8	272.7	1.5e-06	470129	0000/00/05	10:35:29
plate411.bin	15	50	291	9e-06	97829	0000/00/16	23:37:15
plate405.bin	1	57.4	777	7e-06	102462	0000/00/01	04:27:42
plate433.bin	10	17.6	700	0.00071	4280	0000/00/00	11:53:20
plate411.bin	1200	92.44	750	5e-05	14563	0000/06/21	06:20:00
plate422.bin	60	111.3	25	0.0065	2647	0000/00/01	20:07:00
plate433.bin	60	34.7	375	0.0019	1717	0000/00/01	04:37:00
plate405.bin	30	61.31	100	7e-06	91073	0000/01/00	14:56:30
plate422.bin	60	111.3	300	0.0065	96	0000/00/00	01:36:00
plate422.bin	15	22.1	471.75	8.5e-05	7351	0000/00/01	06:37:45
plate433.bin	1	23	400	0.002	1	0000/00/00	00:00:01
